rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy

```diff
@@ -2796,9 +2804,25 @@ class ScheduledExecutionController  extends ControllerBase{
                 }
             }
             if(opt.optionValuesPluginType) {
-                opt.valuesFromPlugin = optionValuesService.getOptions(scheduledExecution.project,opt.optionValuesPluginType)
+                KeyStorageTree storageTree = storageService.storageTreeWithContext(authContext)
+                Map<Class, Object> servicesMap = [:]
+                servicesMap.put(KeyStorageTree, storageTree)
+
+                def services = new RundeckSpiBaseServicesProvider(
+                        services: servicesMap
+                )
```

Comment:
This code seems like it should go in the Option Values service, not in the controller